.env

```diff
@@ -1,6 +1,6 @@
 MYSQL_USER=vitalii
-MYSQL_PASSWORD=1469
-MYSQL_ROOT_PASSWORD=1469
+MYSQL_PASSWORD=14691769
+MYSQL_ROOT_PASSWORD=14691769
 MYSQL_DATABASE=bookstore
 MYSQL_LOCAL_PORT=3307
 MYSQL_DOCKER_PORT=3306

```

Comment:
this file should not be pushed to any open sources. it keeps your secrets. create .env.sample with mock data instead